pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -5290,6 +5290,18 @@ final: prev:
     meta.homepage = "https://github.com/folke/lazy.nvim/";
   };
 
+  lazydev-nvim = buildVimPlugin {
+    pname = "lazydev.nvim";
```

Comment:
```suggestion
    pname = "lazydev-nvim";
```

Reply:
Ah, we should keep lazydev.nvim. It should match the repo name. I'll fix that.